### ISSUE TITLE
New Taxonomy Filter block powered by Interactivity API

### ISSUE THREAD

**Issue by dinhtungdu** [woothemes/woocommerce#42175]

**Comment by dinhtungdu** [comment]
As @nerrad mentioned in a product review, we need to take a closer look at the need before doing any work on it.

**Comment by Marc-pi** [comment]
@dinhtungdu / @nerrad 
any news on this ? as long as i looked for in the issue listing, there is yet no filter by taxonomy in the product filters blocks.
it is a need to close the gap with the classic way (see the remaining options in the customizer)
especially to display subcategories in the category template, or category in the shop template
another option is to offer a way to filter the products by categories, instead of wanting to display a category / sub category grid.
Both capacities are strongly needed, categorization is a base critical feature, and category pages are important landing pages for any eCommerce website.
Related to 
- https://github.com/woocommerce/woocommerce/issues/42241
- https://github.com/woocommerce/woocommerce/issues/42678
- and discussion https://github.com/woocommerce/woocommerce/discussions/57860

**Comment by nerrad** [comment]
cc @ellenbauer to get on your radar.

**Comment by Andrew-Starr** [comment]
There is definitely a need for taxonomy filters IMO.

Please see image below that hopefully explains the need.

![Image](https://github.com/user-attachments/assets/2f61b79c-a6fd-4a9f-abca-f9e77abd62ec)

**Comment by Marc-pi** [comment]
yep, agreed, multi-criteria is a need (combine attributes, price, categories, etc.)
and to manage sub categories as well
